build.cake

```diff
@@ -83,7 +83,7 @@ Task("Build")
 
 Task("Test")
     .IsDependentOn("Build")
-    .WithCriteria(BuildSystem.IsLocalBuild)
+    .WithCriteria(false)
```

Comment:
how does this work, it looks to me like this will never trigger?

Reply:
Ah shoot. I thought I removed this.

Reply:
Reverted. Thanks @shaun-od.